lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscoverySpec.hs

```diff
@@ -13,6 +13,7 @@
 
 module Cardano.Wallet.Primitive.AddressDiscoverySpec
     ( spec
+    , DummyTarget
```

Comment:
Hmmm... Not really sure about this. The test and instance from `AddressDiscovery` are quite specific and tailored for the properties of that module. They only use a single account key for instance. I guess it's okay for the DBSpec, but I'd rather _not_ do that yet, and stick with our current approach of not sharing instances between spec files. This is an open-gate for messiness later. 

We may discuss how we can better share generators between spec files without making spec files depend on each others. It becomes quite hard then to follow which arbitrary instance comes from where....

Reply:
moved to DBspec needed bits, and here export `DummyTarget` to MVarSpec and SliteSpec.